packages/beacon-node/src/eth1/index.ts

```diff
@@ -105,6 +105,10 @@ export class Eth1ForBlockProduction implements IEth1ForBlockProduction {
     this.eth1MergeBlockTracker.startPollingMergeBlock();
   }
 
+  isPollingEth1Data(): boolean | undefined {
+    return this.eth1DepositDataTracker?.isPollingEth1Data();
```

Comment:
this shouldn't return `undefined`

```suggestion
  isPollingEth1Data(): boolean {
    return this.eth1DepositDataTracker?.isPollingEth1Data() ?? false;
```